CONTRIBUTING.md

```diff
@@ -71,31 +71,39 @@ All the provider names of packages and so one are the ones used by the provider
 
-To know what's the ID expected by Terraform you can check on the Terraform documentation of the resource ([doc](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/instance#import)[TODO: link docs and tf code], sometimes is more complex and it needs several elements to build an ID ([doc](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/iam_role_policy#import) and [code](https://github.com/hashicorp/terraform-provider-aws/blob/main/aws/resource_aws_iam_role_policy.go#L171)) so we have to set it on the right format for then be read for Terraform.
+Resources are each of one of the Terraform resources that this specific provider has and can import using TerraCognita. The main idea is that the `provider.Provider.Resources` returns all the instances of one resource. That information has to be fetched from the provider API using the SDK initialized before and for each instance we have to build the Terraform resource ID and return the resource. This then will be used by Terraform to read all the needed data from the provider to load the resource Schema (no need to do anything as this is abstracted into the caller of that function).
 
-As providers have a lot of resources we have created custom sdks on top of the provider sdks that we use to simplify things. Those wrappers are autogenerated from a configuration that is custom for each Provider. We do this so when we have to add new resources is easy, one of the main reasons is to also avoid pagination and always return all the resources directly. For example in `aws` we have the [configuration](https://github.com/cycloidio/terracognita/blob/master/aws/cmd/functions.go) which has the [go:generate](https://github.com/cycloidio/terracognita/blob/master/aws/reader/connector.go#L56) that generates the end [reader](https://github.com/cycloidio/terracognita/blob/master/aws/reader/reader.go). We have similar things for the other sdks
+To know what's the ID expected by Terraform you can check on the Terraform documentation of the resource ([doc](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/instance#import), sometimes is more complex and it needs several elements to build an ID ([doc](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/iam_role_policy#import) and [code](https://github.com/hashicorp/terraform-provider-aws/blob/main/aws/resource_aws_iam_role_policy.go#L171)) so we have to set it on the right format for then be read for Terraform.
 
-The next point on the documentation explains how to add new resources to a provider which will help on doing it from 0 too.
+There are some specific cases in which a resource is imported with another one, on the AWS case that would be `aws_security_groups`, so some times one resource can generate also other resources when then it's pulled by Terraform.
```

Comment:
It was more about the "choice" to be made. Either `security_group` with rules defined inline, or `security_group` + `security_group_rules` associated